pkg/executables/kubectl.go

```diff
@@ -677,6 +677,15 @@ func (k *Kubectl) ValidateEKSAClustersCRD(ctx context.Context, cluster *types.Cl
 	return nil
 }
 
+func (k *Kubectl) SetControllerEnvVar(ctx context.Context, envVar, envVarVal, kubeconfig string) error {
```

Comment:
Should we be providing a mechanism for toggling stuff on post deployment? That seems dangerous because what you deploy originally might not be what's running in the end due to arbitrary logic altering the environment.

Reply:
I expect a secret to be changed because they rotate. Unlike secrets that are typically data, we're toggling behavior with this configuration which seems awfully dangerous to me. @vivek-koppuru do you have any suggestions on how else this could be done? 'easiest' isn't necessarily 'right'.

Reply:
Yea I thought about this but refrained from commenting on it because I couldn't think of a different way without affecting the manifest itself. I think the right way would be to add this condition in our manifest code through our bundle, but that requires more refactoring efforts in terms of scope. I think we should be able to provide a way to edit our deployment env vars in the future, so if we have this as an action item we can address it then. Since this is behind a feature flag and we know that the blast radius is minimized, I am okay with this current approach right now.

Reply:
Lets commit to this. @vivek-koppuru would you be able to raise an issue with some of your ideas?